src/views/Home.vue

```diff
@@ -253,6 +278,17 @@ export default {
         this.pickedHeadline %= Math.min(this.headline.length, 5)
       }, this.HEADLINEINTERVAL)
       this.pickedHeadline = 0
+    },
+    getLoginStatus() {
+      return this.$store.getters.isLogin
+    },
+    getRecommendHashtags() {
+      // TODO: get recommend hashtags by api
+      return [
+        '九合一大選', '強尼戴普', '加密貨幣', 'COVID-19', '墮胎自主權',
+        '防疫政策', '烏俄戰爭', '加密貨幣', '普丁', '快篩試劑',
+        '兒童疫苗', '金曲獎33', '猴痘', '確診'
+      ]
```

Comment:
Write a mock API. This component will not have to be changed later.

Reply:
Done.